cogip/entities/robot.py

```diff
@@ -5,14 +5,15 @@
 from PySide6.QtCore import Slot as qtSlot
```

Comment:
Well, actually this is only what I really did! But I changed a bit the way of doing it ;-)

I change the way of handling the robot order entity (the one that indicate order position).
For the real robot, we use a detailed STEP file with colors, but it is overkill for robot order
entity (in terms of load time, memory consumption and code complexity).
Using simple shape from a OBJ file is much faster and the code is much simpler since it does
not required to load an entire scene and modify.